src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
We did add a bunch of other suggestions that are on by default, but we always had an option for people to turn them off.

I'm not sure we need an option to force explicit names in those cases, but we might want an option for whether we enforce this or not.